backbone.js

```diff
@@ -1851,6 +1851,7 @@
       // Is pushState desired ... is it available?
       this.options          = _.extend({root: '/'}, this.options, options);
       this.root             = this.options.root;
+      this._trailingSlash   = this.options.trailingSlash || false;
```

Comment:
I don't care much about this, but technically, you can afford to leave off the `|| false`.

Reply:
Yah--I find it a helpful hint that this is going to be interpreted as a Boolean; but it is redundant.